app/script/notification/NotificationRepository.js

```diff
@@ -153,25 +152,6 @@ z.notification.NotificationRepository = class NotificationRepository {
     });
   }
 
-  // Remove notifications from the queue that are no longer unread
-  removeReadNotifications() {
-    this.notifications.forEach(notification => {
-      const {conversationId, messageId, messageType} = notification.data || {};
-
-      if (messageId) {
-        this.conversationRepository.is_message_read(conversationId, messageId).then(isRead => {
-          if (isRead) {
-            notification.close();
-            const messageInfo = messageId
-              ? `message '${messageId}' of type '${messageType}'`
-              : `'${messageType}' message`;
-            this.logger.info(`Removed read notification for ${messageInfo} in '${conversationId}'.`);
-          }
-        });
-      }
-    });
-  }
-
```

Comment:
Why do we want to remove this? This ensure a closure of notifcations when a conversation was marked as read on other devices etc.